docs/HOWTO_INTEGRATE_TWILIO.md

```diff
@@ -5,34 +5,36 @@ You will need to create a Twilio account in order to test outgoing and incoming
 
 ## Instructions
 
-### Note
-If you are using these instructions for an Heroku instance or AWS Lambda instance, replace references to <YOUR_NGROK_APP_URL> with your actual app url (i.e. examplespokeapp.herokuapp.com )
+### Note on Using Heroku or AWS Lambda
+> If you are using these instructions for an Heroku instance or AWS Lambda instance, replace references to <YOUR_APP_URL> with your actual app url (i.e. examplespokeapp.herokuapp.com ), and skip step 1.
 
 1. If you are using these instructions for development (not Heroku), you can use [ngrok](https://ngrok.com/docs) to allow Twilio to communicate with Spoke.
   - Create an ngrok account: https://ngrok.com/
   - For a paid ngrok account, a reasonable command line for `ngrok` is `ngrok http -subdomain=<UNIQUE_NAME> 3000` (Replace <UNIQUE_NAME> with something likely to be globally unique. Use the same <UNIQUE_NAME> each time you start ngrok. If you use a different <UNIQUE_NAME> it will be necessary to change the configuration in Twilio and in your .env file.)
   - For a free ngrok account, ngrok can be started with `ngrok http 3000` (Because a free account receives a different, randomly-assigned URL every time ngrok is started, it will be necessary to change the configuration in Twilio and in your .env file each time.)
-  - When you start `ngrok`, it will display the external-facing app URL. Use that below to replace <YOUR_NGROK_APP_URL>
-2. Create a Twilio acccount: https://www.twilio.com/ _If your organization is a non-profit organization, you may also apply for complementary twilio credits at [Twilio.org] (www.twilio.org/application)_
+  - When you start `ngrok`, it will display the external-facing app URL. Use that below to replace <YOUR_APP_URL>
+2. Create a Twilio acccount: https://www.twilio.com/ _If your organization is a non-profit organization, you may also apply for complementary twilio credits at [Twilio.org](www.twilio.org/application)_
 3. Click on `Programmable SMS` on the side panel
-4. Click on `Messaging Services`, and click the plus
-5. Create a Friendly Name and select the `Mixed` Use Case, then click `Create`
-6. Under `Properties`
+4. Click on `Messaging Services`, and click `Create Messaging Service`
+5. Give your messaging service a Name then click `Create`
```

Comment:
we do want to keep the `Mixed` use case instruction unless you've seen that be different?